xla/service/gpu/tests/gpu_replica_reduce_splitter_test.cc

```diff
@@ -0,0 +1,98 @@
+/* Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
The formatting here seems to be off. Can you please copy the header from one of the other files?